pkg/kubelet/kuberuntime/kuberuntime_container_linux_test.go

```diff
@@ -167,13 +167,14 @@ func TestGenerateLinuxContainerConfigResources(t *testing.T) {
 	assert.NoError(t, err)
 
 	tests := []struct {
-		name         string
-		podResources v1.ResourceRequirements
-		expected     *runtimeapi.LinuxContainerResources
+		name               string
+		containerResources v1.ResourceRequirements
+		podResources       *v1.ResourceRequirements
+		expected           *runtimeapi.LinuxContainerResources
 	}{
 		{
 			name: "Request 128M/1C, Limit 256M/3C",
-			podResources: v1.ResourceRequirements{
```

Comment:
Did you replace the test cases with the new ones? Why not keeping the originals too?

Reply:
didn't remove the tests; the test had a field named "podResources" changed to to containerResourced and added tests with podResources relevant for my change

Reply:
NVM. The field just got renamed.